fbw-common/src/systems/instruments/src/ND/pages/rose/RoseLSPage.tsx

```diff
@@ -87,6 +87,23 @@ export class RoseLSPage<T extends number> extends RoseMode<T, RoseLsProps<T>> {
   render(): VNode | null {
     return (
       <g visibility={this.isVisible.map((v) => (v ? 'visible' : 'hidden'))}>
+        {/* inverted map overlays for terrain map in WASM module  */}
+        <path
+          name="rose-mode-bottom-left-map-area"
+          d="M45,625 L122,625 L174,683 L174,768 L0,768 L0,0 L45,0L45,625"
+          class="nd-inverted-map-area"
+        />
+        <path
+          name="rose-mode-bottom-right-map-area"
+          d="M591,768 L591,626 L648,562 L723,562 L723,0 L768,0 L768,768 L591,769"
+          class="nd-inverted-map-area"
+        />
+        <path
+          name="rose-mode-top-map-area"
+          d="M45,0 L45,155, L282,155 a250,250 0 0 1 204,0 L723,155 L723,0 L45,0"
+          class="nd-inverted-map-area"
+        />
+
```

Comment:
Aren't these different between A320 and A380 so they can't go in fbw-common?

Reply:
For ROSE NAV, they are already there. The left and right info area are currently the same for A32NX and A380X. Even though you won't notice them in the A32NX (because the TERR doesn't go that far down), it doesn't hurt to have them here

Reply:
The mask at the top is different though. The A380 mask extends above the rose, whereas the A320 has a cutout extending below the top of the rose except the area inside it.

Reply:
Yes, but removing the mask alone won't fix it, since SimBridge also somewhere has a mask on its own (which doesn't align with the ND roses so it doesn't look clean.
For now, until we can fix the SimBridge mask I prefer to have the top mask for both a/c